Note Ruby incompability with Jekyll for GitHub Pages (#44809)
github/docs: content/pages/setting-up-a-github-pages-site-with-jekyll/testing-your-github-pages-site-locally-with-jekyll.md

   >
   >   To fix the error, try running `bundle add webrick`, then re-running `bundle exec jekyll serve`.
   >

>    >   You might get other errors (related to missing gems and methods) if you've installed Ruby 3.2 or later. It is recommended to install Ruby 3.1.x or earlier instead due to compatibility issues with the `github-pages` gem.

Reviewer: ```suggestion
   >   If you've installed Ruby 3.2 or later, you may see other errors related to missing gems and methods, due to compatibility issues with the `github-pages` gem. In this event, you should install Ruby 3.1.x or earlier instead.
```
